### ISSUE TITLE
Support Windows OS?

### ISSUE THREAD

**Issue by RandomDefaultUser** [mala-project/mala#56]
In GitLab by @RandomDefaultUser on Feb 26, 2021, 19:44

We might have to clarify on what platform our collaborators are developing. I'm not sure if our code works on Windows systems right away, or if there is need to change, e.g. file path issues (its always good practice to use the os independent path module `os.path` from the python standard library anyway, e.g. `os.path.join`).

- [x] check if there is need to support windows
- [x] if so, check if fesl is windows compatible and fix issues if not

**Comment by RandomDefaultUser** [comment]
link_data_repo.sh needs to be replaced by a .bat file for windows.

**Comment by RandomDefaultUser** [comment]
Apart from this replacement @tamar-yovell was able to run MALA on Windows successfully. I'll write a batch script shortly and then we can close this issue.

**Comment by DanielKotik** [comment]
We should also add a `make.bat` alongside `Makefile` in the `docs` folder. We initially used `sphinx-quickstart` to generate a `conf.py` and Makefiles for Linux/Windows. Since I changed the `Makefile` (deleted not used functions, added `apidocs`), I also deleted `make.bat`. There is a tool for conversion of a `Makefile` to its Windows pendant, https://github.com/espositoandrea/Make-to-Batch . We should check, if the generated `make.bat` works and can generate the Sphinx docs locally.
